.vscode/launch.json

```diff
@@ -18,7 +18,7 @@
       // As such, this path would have to change accordingly.
       "program": "${workspaceFolder}/target/debug/madara",
       // If you wish to supply arguments/parameters to the program, supply them below:
-      "args": ["--dev", "--sealing=instant"],
+      "args": ["--dev", "--sealing=instant", "--tmp", "--execution=Native"],
```

Comment:
why are u using native execution?

Reply:
this is for launch.json which is used when using the vs code debugger. the debugger won't work if the execution isn't native